elastic4s-core/src/main/scala/com/sksamuel/elastic4s/update/UpdateByQueryDefinition.scala

```diff
@@ -1,13 +1,13 @@
 package com.sksamuel.elastic4s.update
 
-import com.sksamuel.elastic4s.Indexes
+import com.sksamuel.elastic4s.IndexesAndTypes
 import com.sksamuel.elastic4s.script.ScriptDefinition
 import com.sksamuel.elastic4s.searches.queries.QueryDefinition
 import com.sksamuel.exts.OptionImplicits._
 
 import scala.concurrent.duration.FiniteDuration
 
-case class UpdateByQueryDefinition(sourceIndexes: Indexes,
+case class UpdateByQueryDefinition(indexesAndTypes: IndexesAndTypes,
```

Comment:
Are breaking changes in these inner query definitions OK? What about at minor released as well if I merge it to 5.5.x branch?

Reply:
I'm ok with this changing, but lets add an apply method in a companion object that accepts Indexes and converts it, so it doesn't break anything.